cpp/src/parquet/column_writer.cc

```diff
@@ -1051,7 +1061,9 @@ void ColumnWriterImpl::BuildDataPageV2(int64_t definition_levels_rle_size,
   ConcatenateBuffers(definition_levels_rle_size, repetition_levels_rle_size,
                      compressed_values, combined->mutable_data());
 
-  EncodedStatistics page_stats = GetPageStatistics();
+  EncodedStatistics page_stats;
+  std::shared_ptr<SizeStatistics> page_size_stats;
+  std::tie(page_stats, page_size_stats) = GetPageStatistics();
```

Comment:
Note: you can use the nicer [C++17 structured bindings](https://en.cppreference.com/w/cpp/language/structured_binding)